test/gui/shared/scripts/helpers/api/sharing_helper.py

```diff
@@ -109,3 +109,20 @@ def share_resource(user, resource, receiver, permissions):
     request.assertHttpStatus(
         response, 200, f"Failed to share resource '{resource}' to user '{user}'"
     )
+
+
+def create_link(user, resource, name, password, expireDate, permissions):
```

Comment:
make `user`, `resource` and `permissions` as required param
```suggestion
def create_link(user, resource, permissions, optional_params=None):
```